client/app/account/login/login.html

```diff
@@ -7,3 +7,9 @@
     </a>
   </div>
 </div>
+
```

Comment:
The api_key script tag should be removed, and those values should be stored in local.env.js